tests/testthat/test-utils.R

```diff
@@ -248,9 +271,15 @@ test_that("configure_unixodbc_spark() writes reasonable entries", {
     con = spark_config_path
   )
 
-  configure_unixodbc_spark(
+  expect_snapshot(configure_unixodbc_simba(
+    unixodbc_install = unixodbc_install_path,
+    simba_config = spark_config_path,
+    action = "warn"
+  ))
+  configure_unixodbc_simba(
```

Comment:
Might be missing an `expect_snapshot()` here?

Reply:
Hey @atheriel - thanks for taking a look!
I am not sure I follow - wasn't able to identify the spot where I can throw a snapshot comparison.

Reply:
I might have misinterpreted, sorry.